etc/config/c++.amazon.win32.properties

```diff
@@ -1,13 +0,0 @@
-# amazon win32 settings
-compilers=vc19:vc19_32
-compiler.cl19.name=x64 msvc 19 (64 bit)
-compiler.cl19.exe=etc\scripts\vc19_amd.bat
-compiler.cl19.includeFlag=/I
-compiler.cl19.versionFlag=/?
-compiler.cl19_32.name=x86 msvc 19 (32 bit)
-compiler.cl19_32.exe=etc\scripts\vc19_x86.bat
-compiler.cl19_32.includeFlag=/I
-compiler.cl19_32.versionFlag=/?
-postProcess=
-binaryHideFuncRe=^(_.*|(de)?register_tm_clones|call_gmon_start|frame_dummy)$
-needsMulti=false
```

Comment:
Discussed on GitHub that this file does not actually initialize any compiler, and that the best idea would be to delete it